docs/how_to_guides/example_models.rst

```diff
@@ -0,0 +1,133 @@
+How to Load Example Models
+==========================
+
+In the tutorials and guides of the **respy** documentation, you will frequently 
+encounter *example models*. These are pre-defined models that can be easily accessed
+with just one function call to facilitate an introductory workflow and use in
+explanatory material. The set of examples consists of simple as well as very advanced
+models to cover a wide range of use cases and challanges that come with an increasing
+degree of complexity in dynamic discrete choice models.
+
+For instance, in the tutorials you will encounter two very simple models that are based 
+on the story of Robinson Crusoe. Other example models are based on actual economic
+publications and thus serve to illustrate the scalability of `respy` models.
+
+You can access example models by typing 
+
+.. code-block:: python
+
+    params, options, data = rp.get_example_model("model_name")
+
+Below are the example models that are currently available. 
+
+
+Toy models
+----------
+
+The package provides users with two toy models. These models can be used to acquaintance
+oneself with **respy**'s functionalities and can be encountered in the tutorials.
+
+.. raw:: html
+
+     <div
+      <p class="d-flex flex-row gs-torefguide">
+          <span class="badge badge-info">To Tutorials</span></p>
+      <p>Check out the <a
+         href="../tutorials/index.html">Tutorials</a> </p>
+     </div>
+
+
+The models are called:
+
+- ``robinson_crusoe_basic``
+- ``robinson_crusoe_extended``
+
+These models are excellent examples to use for learning and prototyping because they are
+very computationally feasible due to the small number of available choices and low
+number of periods in the models.
```

Comment:
These models are excellent examples to use for learning and prototyping: They include a small number of available choices and a low number of periods such that they are computationally feasible.

Reply:
That sounds much better, thanks!